pt-BR/mail/chrome/messenger/focusPane.dtd

```diff
@@ -1,73 +1,73 @@
 <!-- focus pane favorite contacts context menus -->
-<!ENTITY pbFavoriteContactsContextMenu-delete.label     "Remove From Favorites">
+<!ENTITY pbFavoriteContactsContextMenu-delete.label     "Remover dos favoritos">
 <!ENTITY pbFavoriteContactsContactsMenu-delete.accesskey "R">
-<!ENTITY pbFavoriteContactsContextMenu-email.label       "Compose">
+<!ENTITY pbFavoriteContactsContextMenu-email.label       "Criar">
 <!ENTITY pbFavoriteContactsContactsMenu-email.accesskey  "e">
-<!ENTITY pbFavoriteContactsContextMenu-edit.label        "Edit Contact...">
+<!ENTITY pbFavoriteContactsContextMenu-edit.label        "Editar contato...">
 <!ENTITY pbFavoriteContactsContactsMenu-edit.accesskey   "d">
-<!ENTITY pbFavoriteContactsContextMenu-createfilter.label "Create Filter...">
+<!ENTITY pbFavoriteContactsContextMenu-createfilter.label "Criar filtro...">
 <!ENTITY pbFavoriteContactsContactsMenu-createfilter.accesskey "C">
 
-<!ENTITY pbFavoriteContacts.DragLabel "Drag your favorite contacts here">
+<!ENTITY pbFavoriteContacts.DragLabel "Arraste seus contatos favoritos aqui">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY focusPaneSocialAttribute.label            "Social">
-<!ENTITY pbFavoriteContactsContextMenu-openInNewTab.label "Open in New Tab">
+<!ENTITY pbFavoriteContactsContextMenu-openInNewTab.label "Abrir em nova aba">
 <!ENTITY pbFavoriteTopicsContextMenu-openInNewTab.accesskey "N">
 <!ENTITY pbFavoriteContactsContextMenu-openInNewTab.accesskey "N">
-<!ENTITY focusPaneNotToDoAttribute.label           "Not Reminders">
-<!ENTITY focusPaneRepliedAttribute.label           "Replied">
+<!ENTITY focusPaneNotToDoAttribute.label           "Não são lembretes">
+<!ENTITY focusPaneRepliedAttribute.label           "Respondidos">
 <!ENTITY pbFavoriteContactsContextMenu-edit.accesskey   "d">
-<!ENTITY focusPaneToDoAttribute.label              "Reminders">
-<!ENTITY pbFavoriteTopicsContextMenu-openInNewTab.label "Open in New Tab">
+<!ENTITY focusPaneToDoAttribute.label              "Lembretes">
+<!ENTITY pbFavoriteTopicsContextMenu-openInNewTab.label "Abrir em nova aba">
 <!ENTITY pbFavoriteContactsContextMenu-createfilter.accesskey "F">
 <!ENTITY pbFavoriteTopicsContextMenu-assignColor.accesskey "S">
-<!ENTITY focusPaneNotRepliedAttribute.label           "Not Replied">
-<!ENTITY focusPaneNotPendingAttribute.label        "Not Pending">
+<!ENTITY focusPaneNotRepliedAttribute.label           "Não respondidos">
+<!ENTITY focusPaneNotPendingAttribute.label        "Não está pendente">
 <!ENTITY pbFavoriteContactsContextMenu-delete.accesskey "R">
-<!ENTITY focusPaneSubscriptionsAttribute.label     "Subscriptions">
-<!ENTITY focusPanePendingAttribute.label           "Pending">
-<!ENTITY focusPaneToMeOnlyAttribute.label          "To Me Only">
-<!ENTITY pbFavoriteTopicsContextMenu-assignColor.label  "Set Topic Color">
-<!ENTITY focusPaneReadAttribute.label           "Read">
+<!ENTITY focusPaneSubscriptionsAttribute.label     "Assinaturas">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY focusPaneSubscriptionsAttribute.label     "Subscrições">